osf/models/registrations.py

```diff
@@ -479,6 +481,8 @@ class DraftRegistration(ObjectIDMixin, BaseModel):
     #     'value': <value>
     #   }
     # }
+    registration_responses = DateTimeAwareJSONField(default=dict, blank=True)
+    registration_responses_migrated = models.NullBooleanField(default=False)
```

Comment:
Added `registration_responses_migrated` on DraftRegistration and Registration models to track which ones have been migrated over via management command.